src/Pact/Native.hs

```diff
@@ -97,17 +98,17 @@ moduleToMap = M.fromList . map (((`Name` def) . asString) *** Direct) . snd
 
 lengthDef :: NativeDef
 lengthDef = defRNative "length" length' (funType tTyInteger [("x",listA)])
- "Compute length of X, which can be a list, a string, or an object.\
- \`(length [1 2 3])` `(length \"abcdefgh\")` `(length { \"a\": 1, \"b\": 2 })`"
+ ["(length [1 2 3])", "(length \"abcdefgh\")", "(length { \"a\": 1, \"b\": 2 })"]
```

Comment:
A little bikesheddy but wondering why you put the example first, it's less intuitive from a code-reading standpoint to jump straight to examples.